src/auth/awsclient.ts

```diff
@@ -197,10 +218,21 @@ export class AwsClient extends BaseExternalAccountClient {
     );
   }
 
+  private async getAwsSessionToken(): Promise<string> {
+    const opts: GaxiosOptions = {
+      url: this.awsSessionTokenUrl,
+      method: 'PUT',
+      responseType: 'text',
+      headers: {'x-aws-ec2-metadata-token-ttl-seconds': '21600'},
```

Comment:
I think `21600` seconds is a long time. Since we don't cache these responses and regenerate signed requests on demand, is there a need to make it this long?

Reply:
Discussed offline and decided to go with 300 seconds (5 mins)